### PR TITLE
add literature gomez_2003_effect_228

### PR DESCRIPTION
**Pull request by TheOpenBIts**

### CODE REVIEW

File: literature/gomez_2003_effect_228/gomez_2003_effect_228.bib [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@article{gomez_2003_effect_228,
+  title={Effect of temperature on hydrogen adsorption on Pt (111), Pt (110), and Pt (100) electrodes in 0.1 M HClO4},
+  author={G{\'o}mez, Roberto and Orts, Jos{\'e} M and {\'A}lvarez-Ruiz, Bernab{\'e} and Feliu, Juan M},
+  journal={The Journal of Physical Chemistry B},
+  volume={108},
+  number={1},
+  pages={228--238},
+  year={2004},
+  publisher={ACS Publications}
+}
```
Comment (DunklesArchipel) [inline literature/gomez_2003_effect_228/gomez_2003_effect_228.bib:10]:
```suggestion
}

```

File: literature/gomez_2003_effect_228/gomez_2003_effect_228.bib [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@article{gomez_2003_effect_228,
+  title={Effect of temperature on hydrogen adsorption on Pt (111), Pt (110), and Pt (100) electrodes in 0.1 M HClO4},
```
Comment (DunklesArchipel) [inline literature/gomez_2003_effect_228/gomez_2003_effect_228.bib:2]:
```suggestion
  title={Effect of temperature on hydrogen adsorption on Pt(111), Pt(110), and Pt(100) electrodes in 0.1 M HClO4},
```

File: literature/gomez_2003_effect_228/gomez_2003_effect_228_p1_f1_293.svg
Comment (DunklesArchipel) [inline literature/gomez_2003_effect_228/gomez_2003_effect_228_p1_f1_293.svg:1]: The figure label is missing in the figure.

File: literature/gomez_2003_effect_228/gomez_2003_effect_228_p1_f1_293.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      type: RHE
+    counter electrode:
+      material: Pt
+      shape: coiled wire
+    working electrode:
+      material: Pt
+      crystallographic orientation: '111'
+      preparation procedure:
+        description:
+          - The electrode was heated in a gas/air flame and cooled in a hydrogen + argon atmosphere. Adsorption of impurities during the transfer of the sample to the controlled atmosphere of the cell was avoided by protecting the surface with a droplet of ultrapure water in equilibrium with the cooling atmosphere. Contact with the electrolyte solution was made at a controlled potential (0.10 V). Voltammetric experiments were carried out in a hanging meniscus configuration (dipping technique)."
```
Comment (DunklesArchipel) [inline literature/gomez_2003_effect_228/gomez_2003_effect_228_p1_f1_293.yaml:58]: Is this the original text? We should avoid that.
Maybe separate it in steps
```yaml
- heated in gas/air flame
- cooled in H2+Ar
- transfer with the surface protected by a droplet of ultrapure water
- immersed under potential control
```

The hanging meniscus part can be omitted. That is an interesting aspect that should be included somewhere else in the metadata.

File: literature/gomez_2003_effect_228/gomez_2003_effect_228_p2_f2_293.svg
Comment (DunklesArchipel) [inline literature/gomez_2003_effect_228/gomez_2003_effect_228_p2_f2_293.svg:1]: figure label missing

File: literature/gomez_2003_effect_228/gomez_2003_effect_228_p2_f2_293.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    reference electrode:
+      type: RHE
+    counter electrode:
+      material: Pt
+      shape: coiled wire
+    working electrode:
+      material: Pt
+      crystallographic orientation: '100'
+      preparation procedure:
+        description:
```
Comment (DunklesArchipel) [inline literature/gomez_2003_effect_228/gomez_2003_effect_228_p2_f2_293.yaml:57]: see above

File: literature/gomez_2003_effect_228/gomez_2003_effect_228_p2_f3_293.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    reference electrode:
+      type: RHE
+    counter electrode:
+      material: Pt
+      shape: coiled wire
+    working electrode:
+      material: Pt
+      crystallographic orientation: '110'
+      preparation procedure:
+        description:
```
Comment (DunklesArchipel) [inline literature/gomez_2003_effect_228/gomez_2003_effect_228_p2_f3_293.yaml:57]: see above

**Review by DunklesArchipel — CHANGES_REQUESTED** [review]
here are some issues with the text and possible missing labels in the svg

File: literature/gomez_2003_effect_228/gomez_2003_effect_228_p1_f1_293.yaml (hunk 3 above)
Comment (TheOpenBIts) [reply literature/gomez_2003_effect_228/gomez_2003_effect_228_p1_f1_293.yaml]: I have adopted the description of the preparation procedure.

**Review by DunklesArchipel — APPROVED** [review]

**Pull request merged**